src/components/Buy/BuySign.js

```diff
@@ -75,21 +70,8 @@ class BuySign extends React.Component {
 
 	  return (
       <FormWrapper>
-        <InfoRow>
-          <FormInfoText>Price</FormInfoText>
-          <div>{price} {collateralType}</div>
-        </InfoRow>
-        <InfoRow>
-          <FormInfoText>{infotext}</FormInfoText>
-          <div>{priceToBuy} {collateralType}</div>
-        </InfoRow>
-        <InfoRow>
```

Comment:
Why are we removing the Price/Pay Amount/Receive numbers?

Reply:
In the designs this is not showed when is not connected. And thats the section that appears when is not conencted.

Reply:
Oh, I thought this was the section that shows while you are signing a transaction for a buy